src/client/components/game/GameComponent.tsx

```diff
@@ -49,18 +55,7 @@ const GameComponent = () => {
 
   return (
     <div className="game">
-      <div id="game-ui" className="game__intro padding_s-3">
-        <Button onClick={startGame} size="large" className="margin_s-2" styleType="primary">Start game!</Button>
-        <Button
-          onClick={handleFullscreenClick}
-          size="large"
-          className="margin_s-2"
-          styleType="primary"
-        >
-          Start in fullscreen!
-        </Button>
-      </div>
-      <div id="game-score" className="h3 game__score">0</div>
+      <GameUI {...{ startGame, handleFullscreenClick }} />
```

Comment:
Ты деструктуризацию делаешь чтобы пропсы-значения не писать повторяя дважды одно и то же слово? Или почему так?
Такая конструкция выглядит привычнее и читабельнее на мой взгляд, но это мелочь ) 
```suggestion
      <GameUI startGame={startGame} handleFullscreenClick={handleFullscreenClick} />
```